internal/job/integration/executor_tester.go

```diff
@@ -135,6 +135,7 @@ func NewExecutorTester(ctx context.Context) (*ExecutorTester, error) {
 			"PATHEXT="+os.Getenv("PATHEXT"),
 			"TMP="+os.Getenv("TMP"),
 			"TEMP="+os.Getenv("TEMP"),
+			"USERPROFILE="+os.Getenv("USERPROFILE"),
```

Comment:
I spent a bit of time scratching my head over why this still didn't work. Turns out the tester creates a temp `homeDir` and sets `HOME` above to that. We should use the same path for `USERPROFILE`.

```suggestion
			"USERPROFILE="+homeDir,
```